src/core/renderers/canvas/CanvasRenderer.js

```diff
@@ -113,6 +113,13 @@ function CanvasRenderer(width, height, options)
         worldAlpha: 1
     };
 
+    /**
+     * If set to true, requests the mouse cursor lock inside the canvas.
+     *
+     * @member {boolean}
+     * @default false
+     */
+    this.mouseLock = false;
```

Comment:
As this variable is in both the CanvasRenderer and WebGLRenderer; perhaps place it in the SystemRenderer base class.